lib/aeuis.js

```diff
@@ -0,0 +1,39 @@
+var Client = require("./client");
+var AEUIS_PATH = "aeuis";
+
+function Aeuis(){
+}
+
+Aeuis.list = function(client, query, callback){
+    if(arguments.length === 2){
+        callback = query;
+        query = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(AEUIS_PATH), query, function(err, res){
+        if(err){
+            return callback(err);
+        }
+        res.client = client;
+        res.__proto__ = Aeuis.prototype;
+        callback(null, res);
+    });
+};
+
+Aeuis.get = function(client, acid, callback){
+    if(arguments.length === 2){
+        callback = acid;
+        acid = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(AEUIS_PATH) + `/${acid}`, function(err, res){
```

Comment:
Might as well do
```
`${client.concatAccountPath(AEUIS_PATH)}/${acid}`
```

Reply:
Again the template string would be better but just to keep the code looking similar we went with concatenation.  These are all great points!

This SDK was written a while ago.

Reply:
Then why did you use the sting template for `/${acid}`? Idk you might as well do the who thing.